README.md

```diff
@@ -1,9 +1,137 @@
 # scala-rules
-A rule engine supporting forward chaining
+Scala-Rules is a forward chaining rule engine, built in [Scala](https://www.scala-lang.org). The engine comes with a DSL with which derivations can be modelled as small steps that combine into large derived networks of facts.
 
-[ ![Codeship Status for scala-rules/scala-rules](https://codeship.com/projects/628dece0-e3e8-0133-a9e8-3aa3f222b1f1/status?branch=master)](https://codeship.com/projects/146192)
+The project started with a Dutch DSL to facilitate bankers writing mortgage-related calculations. An English translation of the DSL is still in progress.
 
-# Origin and Status
-The scala-rules engine and DSL are a product of a project to allow Business Analysts to write executable specifications of financial calculations.
+[![Codeship Status for scala-rules/scala-rules](https://codeship.com/projects/628dece0-e3e8-0133-a9e8-3aa3f222b1f1/status?branch=master)](https://codeship.com/projects/146192)
 
-The accompanying DSL is currently only available in Dutch, due to the focus of the PoC. Expect an English equivalent to be available soon.
\ No newline at end of file
+# Getting Started
+
+The minimal requirement to get started is to add the artefact containing the rule engine to your project. Below are snippets for SBT and Maven. Be sure to check if any newer versions exist and adjust the snippet accordingly.
+
+**SBT** - `build.sbt`
+```
+libraryDependencies ++= "org.scala-rules" %% "rule-engine" % "0.2.6"
+```
+
+**Maven** - `pom.xml`
+```xml
+<dependency>
+	<groupId>org.scala-rules</groupId>
+	<artifactId>rule-engine_2.11</artifactId>
+	<version>0.2.6</version>
+</dependency>
+```
+
+# Concepts
+
+The rule engine uses a `Fact` as the base for all of its derivations. A `Fact` defines its name and type so it can be referenced in evaluations down the road. `Facts` are best defined inside a `Glossary`:
+
+```scala
+class MyGlossary extends Glossary {
+	val factA = defineFact[Int]("factA")
+	val factB = defineFact[Int]("factB")
+	val factC = defineFact[Int]("factC")
+}
+```
+
+*Note: the name of the fact is currently repeated as the first argument of the `defineFact` function. This string is actually used inside the engine and must be unique for all `Facts` used by the engine. We are looking for ways to remove this parameter, if you have any suggestions, let us know.*
+
+Using this glossary, it is now possible to define derivations. The Scala Rules DSL (currently only in Dutch) provides an easy way to express how facts interact come together to form your logic. To enable the DSL, create a class that extends `Berekening`:
```

Comment:
"..express how facts interact come together to form your logic.." this is missing an "and" or something similar between the "interact" and "come"